server/game/cards/characters/05/serlancellannister.js

```diff
@@ -0,0 +1,25 @@
+const DrawCard = require('../../../drawcard.js');
+
+class SerLancelLannister extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.getOtherLannisterLords().length == 1,
+            match: this,
+            effect: ability.effects.dynamicStrenght(() => this.getOtherLannisterLords().first().getStrength())
+        });
+    }
+
+    getOtherLannisterLords() {
```

Comment:
Can this method do the logic for checking that there is only one other, and if there is, return it, and if there is not, return undefined?

Also can its name be something like getSingleOtherLannisterLordOrLady ?